priority-queue/PriorityQueueSender/local.settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "WEBSITE_MAX_DYNAMIC_APPLICATION_SCALE_OUT": 200,
+    "TopicName": "topic_1",
+    "ServiceBusConnectionString": "Endpoint=sb://mgransb1.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=oDJmLeOLWFI9SgTCLW72WTN+CNV2ZL3tsCN+vEUdw/Q="
```

Comment:
Probably need to address this ...

Also, once you've got this with managed identity, it won't matter.  Please rotate your keys or recreate this service to prevent fraud/abuse.

Reply:
Addressed